projects/docs/docs/plugins/using-plugins.md

```diff
@@ -11,7 +11,7 @@ You may use plugins that exist at a path local to your machine or at a git sourc
 
 ### Adding plugins
 
-In order to tell Tuist you'd like to use plugins in your project, you can start by defining the plugins in the [`Config.swift`](manifests/config.md) manifest.
+In order to tell Tuist you'd like to use plugins in your project, you can start by defining the plugins in the [`Config.swift`](https://tuist.github.io/tuist/latest/documentation/projectdescription/config) manifest.
```

Comment:
this is one of the examples where I would not want to be redirected to a completely new website - for these links, we should try to stay in the context of current documentation and redirect users to docc only for detailed description of the API.